packages/playground/src/components/pull-request/pull-request-changes.tsx

```diff
@@ -111,7 +119,7 @@ export function PullRequestChanges({ data }: DataProps) {
   return (
     <div className="flex flex-col gap-4">
       {data.map((item, index) => (
-        <PullRequestAccordion key={index} header={item} data={item?.data} />
+        <PullRequestAccordion key={`${item?.title}-${index}`} header={item} data={item?.data} />
```

Comment:
This can potentially generate keys such as `-0`, `-1`, `-2` if title is empty/undefined.

Reply:
Do I do a null check then so that doesn’t happen ? @vardanbansal-harness

Reply:
added null check